core/tests/protractor/learnerFlow.js

```diff
@@ -118,11 +121,13 @@ describe('Learner dashboard functionality', function() {
       if (browser.isMobile) {
         await adminPage.reloadExploration('exploration_player_test.yaml');
       } else {
+        // Set expectWelcomeModal: true.
```

Comment:
I'm still seeing comments like these in the code. Can you remove them? See my previous comment for why

Reply:
Do you expect me to remove it from everywhere ?. I believed that you pointed  out to particular function @U8NWXD

Reply:
Done! @U8NWXD